### ISSUE TITLE
[Feature] Jump to coordinates

### ISSUE THREAD

**Issue by fabioneves** [jabbink/PokemonGoBot#750]
Would be cool if we could insert in WebUI a jump to coordinates without needing to stop the bot.
If we have coordinates of some rare pokemon it's slow to do all of this manually.

**Comment by etkstim** [comment]
Nice feature !

**Comment by CosmoRing** [comment]
We already have a similar function. You just need to righ click on the map and it will add a point (hand pointing down) to the map. After the bot finishes its current task it will walk to the point added.

**Comment by smileytechguy** [comment]
@CosmoRing @fabioneves means for it to just jump, or have a faster speed to just jump to a location

**Comment by fabioneves** [comment]
Exactly. Imagine I'm in the USA, jumping to Europe would take ages, if I just "right click" on a map and I would lose the spawn that I want.

**Comment by CosmoRing** [comment]
I understand, increasing the speed but jumping from USA to Europe will only get you softban. Then you can get that spawn you want.

**Comment by fabioneves** [comment]
It's fine as long as we don't jump to the pokemon exact spawn, just jump near a pokestop, it'll unban then we right click on the pokemon spawn, so the bot goes there to catch it.

**Comment by pashorizer** [comment]
Yeah, until they fix the softban bypass glitch.

**Comment by fabioneves** [comment]
Yup.

**Comment by pashorizer** [comment]
So its pointless.
